get-es-instance-count.sh

```diff
@@ -13,7 +13,7 @@
 BRANCH=$1
 
 if [[ $BRANCH == 'develop' ]] ; then
-  echo "1"
```

Comment:
@mmarcotte, note this change - we should be able to update this config for the court repo's needs if this isn't desired.